src/main/resources/messages.properties

```diff
@@ -55,3 +60,26 @@ get.group.error.title=Get group failed.
 get.group.error.message=Something went wrong while reading the group with id {0}.
 get.group.notFound.title=Group not found.
 get.group.notFound.message=Group with id {0} not found.
+
+create.transaction.error.title=Create transaction failed.
+create.transaction.error.message=Something went wrong while creating the transaction {0}.
+create.transaction.illegalArgument.title=Invalid data.
+create.transaction.illegalArgument.nullOrEmpty.transaction.message=The transaction must not be null nor be empty.
+create.transaction.illegalArgument.nullOrEmpty.name.message=The transaction name must not be null nor be empty.
+create.transaction.illegalArgument.name.tooLong.message=The transaction name must not exceed {0} characters.
+create.transaction.illegalArgument.nullOrEmpty.currency.message=The transaction currency type must not be null nor be empty.
+create.transaction.illegalArgument.currency.tooLong.message=The transaction currency type must not exceed {0} characters for its code.
+create.transaction.illegalArgument.nullOrEmpty.creditor.id.message=The creditor id must not be empty.
+create.transaction.illegalArgument.nullOrEmpty.debtor.id.message=The debtor id must not be empty.
+create.transaction.illegalArgument.nullOrEmpty.createdBy.id.message=The debtor id must not be empty.
```

Comment:
Este mensaje deberia decir "The createdBy id must not be empty"

Reply:
Thanks, no vi esto, igual cambie un poco el mensaje por si lo quieres ver